Assets/LeapMotion/Scripts/Utils/FingerType.cs

```diff
@@ -0,0 +1,38 @@
+﻿
+namespace Leap {
+
+  public static class FingerType {
+
+    public static Finger.FingerType Thumb {
+      get {
+        return Finger.FingerType.TYPE_THUMB;
+      }
+    }
+
+    public static Finger.FingerType Index {
+      get {
+        return Finger.FingerType.TYPE_INDEX;
+      }
+    }
+
+    public static Finger.FingerType Middle {
+      get {
+        return Finger.FingerType.TYPE_MIDDLE;
+      }
+    }
+
+    public static Finger.FingerType Ring {
+      get {
+        return Finger.FingerType.TYPE_RING;
+      }
+    }
+
+    public static Finger.FingerType Pinky {
+      get {
+        return Finger.FingerType.TYPE_PINKY;
+      }
+    }
+
+  }
+
+}
```

Comment:
No, it turns `Finger.FingerType.TYPE_PINKY` into `FingerType.Pinky`(Leap namespace, not a member of Finger).
Dunno about autocomplete, but if you're typing FingerType, it should only show you the one in your namespace, so you'd only be seeing both if you were currently inside the Finger class (which is very rare).
The core idea is just to allow this behavior before the DLL changes but, I can also just remove this and we can just get the DLL changes in (for FingerType.Foo) and none of this FingerType stuff matters anyway.